src/kt_object.cpp

```diff
@@ -1,3 +1,8 @@
 #include "kt_object.h"
 
-KtObject::KtObject(jni::JObject p_wrapped, jni::JObject p_class_loader) : JavaInstanceWrapper("godot.core.KtObject", p_wrapped, p_class_loader) {}
+KtObject::KtObject(const jni::JObject &p_wrapped, const jni::JObject &p_class_loader, jni::JObject *p_ktClass)
```

Comment:
`jni::JObject *p_ktClass` shouldn't be a pointer, `jni::JObject p_ktClass` should suffice. `JObject` just wraps `jobject` which is a pointer itself.